src/components/sbb-tooltip-trigger/sbb-tooltip-trigger.tsx

```diff
@@ -31,15 +31,15 @@ export class SbbTooltipTrigger implements ComponentInterface, ButtonProperties {
   @Element() private _element!: HTMLElement;
 
   private _handlerRepository = new HandlerRepository(this._element, actionElementHandlerAspect);
-  private _isInFormField = false;
 
   public connectedCallback(): void {
     this._handlerRepository.connect();
-    this._isInFormField = !!(
+    if (
       hostContext('sbb-form-field', this._element) ??
-      hostContext('[data-form-field]', this._element) ??
-      this._element.dataset.iconSmall === ''
-    );
+      hostContext('[data-form-field]', this._element)
+    ) {
+      this._element.dataset.iconSmall = '';
```

Comment:
I like the solution now :) Although it's the same, I'd prefer to use toggleDatasetEntry helper here.

``

Reply:
But toggleDatasetEntry removes the attribute if the condition is not true, we just want to set it in case it's true, unless we want to include the condition `this._element.dataset.iconSmall === ''` I don't think this utility is what we need  for this case